### ISSUE TITLE
Add Java 8 support to images?

### ISSUE THREAD

**Issue by gdelangel** [jupyter/docker-stacks#402]
Hi! I was wondering if there were any plans on supporting Java 8 on these images, or if someone had added Java 8 successfully? 
Use case: I'm playing with Hail (hail.is) as a Spark-based framework to handle genomic variant data, and its libraries require Java 8.
Executing for example docker run -it --rm -p 8888:8888 jupyter/all-spark-notebook bash
and then "java -version" yields 
java version "1.7.0_111"
OpenJDK Runtime Environment (IcedTea 2.6.7) (7u111-2.6.7-1~deb8u1)
OpenJDK 64-Bit Server VM (build 24.111-b01, mixed mode)

**Comment by parente** [comment]
Since Spark 2.0 landed in the images, they've contained Java 1.8. Make sure you have a recent version of the images.

```
docker pull jupyter/all-spark-notebook
docker run -it --rm jupyter/all-spark-notebook bash
jovyan@b080fc203121:~/work$ java -version
openjdk version "1.8.0_131"
OpenJDK Runtime Environment (build 1.8.0_131-8u131-b11-1~bpo8+1-b11)
OpenJDK 64-Bit Server VM (build 25.131-b11, mixed mode)
```

**Comment by parente** [comment]
I'm going to close this issue as answered. @gdelangel please reopen it if you still cannot locate Java 8 after updating to the latest image.
